server/bundles/io.cloudbeaver.server/src/io/cloudbeaver/service/WebServiceBindingBase.java

```diff
@@ -121,14 +120,20 @@ protected static WebConnectionInfo getWebConnection(DataFetchingEnvironment env)
         return getWebConnection(getWebSession(env), getProjectReference(env), env.getArgument("connectionId"));
     }
 
+    /**
+     * Returns WebSession or null if the session was not found in the cache, the session could not be restored, or an error occurred
+     */
+    @Nullable
     public static WebSession findWebSession(DataFetchingEnvironment env) {
```

Comment:
Are we sure we want to change this behavior?
getOrRestoreSession creates a new session while findWebSession mustn't do it (it was added specifically to avoid new session creation).

Reply:
We have getWebSession instead. It may create new session.

Reply:
`getOrRestoreSession` does not create a new session.
It takes the session from the cache, if not there, it tries to restore it. If the restore failed, null is returned.
When restoring a session, CBEmbeddedSecurityController#findTokenByAppSession checks that the session being restored has not expired, so we will not get an invalid session

Reply:
findWebSession mustn't restore session. It is used only in a few places specifically to avoid any session refresh/restore/etc. E.g. when we read error log. 

Changing its behaviour may break applicationm workflow. Why do we need in this ticker, how is it related to web sockets?